pkg/pdutil/pd.go

```diff
@@ -170,7 +170,12 @@ func NewPdController(
 		return nil, errors.Annotatef(berrors.ErrPDUpdateFailed, "pd address (%s) not available, please check network", pdAddrs)
 	}
 
-	version := semver.New(string(versionBytes))
+	version, err := semver.NewVersion(string(versionBytes))
+	if err != nil {
+		log.Warn("fail back to old version",
+			zap.Binary("version", versionBytes), zap.Error(err))
+		version = pauseConfigVersion
```

Comment:
looks strange that `pauseConfigVersion` have to become an exclusive bound because of this line. 

and the fallback version should be v0.0.0?